<launch>
    <node name="hdt_arm_teleop" pkg="hdt" type="hdt_arm_teleop" output="screen">
        <rosparam command="load" file="$(find rcta)/config/hdt_arm_teleop.yaml"/>
        <param name="mappings_fname" value="$(find rcta)/config/mappings.lua"/>
    </node>
</launch>
